csharp/ql/integration-tests/all-platforms/standalone/test.py

```diff
@@ -0,0 +1,3 @@
+from create_database_utils import *
+
+run_codeql_database_create([], lang="csharp", extra_args=["--extractor-option=buildless=true", "--extractor-option=cil=false"])
```

Comment:
recently I changed `run_codeql_database` to have `[]` as default command, so to invoke the autobuilder you can simply write
```suggestion
run_codeql_database_create(lang="csharp", extra_args=["--extractor-option=buildless=true", "--extractor-option=cil=false"])
```